packages/devextreme/js/__internal/ui/splitter/resize_handle.ts

```diff
@@ -78,19 +78,27 @@ class ResizeHandle extends (Widget as any) {
       case 'next':
         return `${RESIZE_HANDLE_COLLAPSE_NEXT_PANE_CLASS} ${ICON_CLASS} ${this._getCollapseIconClass(true)}`;
       case 'icon':
+        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} ${this._getResizeIconClass()}`;
       default:
-        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} dx-icon-overflow`;
+        return `${RESIZE_HANDLE_ICON_CLASS} ${ICON_CLASS} .dx-icon-overflow`;
     }
   }
 
+  _getResizeIconClass(): string {
+    const isHorizontal = this._isHorizontalDirection();
+    console.log(isHorizontal);
```

Comment:
Seems, I forgot to remove the console.

Reply:
ok sir will change this one

Reply:
correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26878/commits/6754eeced384460fb394c2160b6e5d628e935f80)